src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -168,3 +173,102 @@ String InstanceName
 //    colocate: Optional[bool]
 //    ): EntityBase();
 
+
+public record AzureSecurityExtensionConfig();
+public record GenevaExtensionConfig();
+
+
+public record KeyvaultExtensionConfig(
+    string KeyVaultName,
+    string CertName,
+    string CertPath,
+    string ExtensionStore
+);
+
+public record AzureMonitorExtensionConfig(
+    string ConfigVersion,
+    string Moniker,
+    string Namespace,
+    [property: JsonPropertyName("monitoringGSEnvironment")] string MonitoringGSEnvironment,
+    [property: JsonPropertyName("monitoringGCSAccount")] string MonitoringGCSAccount,
+    [property: JsonPropertyName("monitoringGCSAuthId")] string MonitoringGCSAuthId,
+    [property: JsonPropertyName("monitoringGCSAuthIdType")] string MonitoringGCSAuthIdType
+);
+
+public record AzureVmExtensionConfig(
+    KeyvaultExtensionConfig? Keyvault,
+    AzureMonitorExtensionConfig AzureMonitor
```

Comment:
In the python code, `AzureSecurityExtensionConfig` and `GenevaExtensionConfig` are both defined models included in `AzureVmExtensionConfig`. These are later used to check to include the extensions or not based off the instance_config passed to the instance. 

How will this happen in the future and what was the thinking behind leaving out these two definitions.

Reply:
For now idea is just to port 1 to 1

If you have any ideas, do take notes and think about them. We'll discuss all those strategies once we have 1 to 1 port complete and we start re-architecture phase